perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -612,7 +613,14 @@ sub garden_waste_cost_pa {
 
     $bin_count ||= 1;
 
-    return $self->feature('payment_gateway')->{ggw_cost} * $bin_count;
+    my $cost = $self->feature('payment_gateway')->{ggw_cost} * $bin_count;
+    my $now = DateTime->now( time_zone => FixMyStreet->local_time_zone );
+
+    if ($now->month =~ /^(10||11||12)$/ ) {
```

Comment:
You only want single `|` here, otherwise the empty string could match (I realise it wouldn't)